docs/tex/tutorials/probabilistic-pca.tex

```diff
@@ -0,0 +1,87 @@
+\newcommand{\R}{ \mbox{${\Bbb R}$}}
+
+
+\title{Probabilistic PCA}
+
+\subsection{Probabilistic PCA}
+
+Probabilistic PCA is useful when dealing with data that is dependent on both principal axes and latent variables \citep{Tipping99probabilisticprincipal}. It is often used when there are missing values in the data.
+
+We'll run through a quick tutorial to implement it in Edward below. The full script can be accessed
+\href{https://github.com/blei-lab/edward/blob/master/examples/probabilistic_pca.py}
+{here}.
+
+\subsubsection{Data}
+
+We simulate our data points below. We'll talk about the individual variables and what they stand for in the next section. For this example, $\mathbf{x}\in\mathbb{R}^2$.
```

Comment:
"For this example, each data point is 2-dimensional, $\mathbf{x}_n\in\mathbb{R}^2$."